docs/export.md

```diff
@@ -105,6 +107,8 @@ optional arguments:
 1. model-name: required, prefix of exported model archive file.
 2. model-path: required, directory which contains files to be packed into exported archive.
 
+**Optional Arguments**
+1. from-type: optional, model type to be packed into model archive
```

Comment:
Please apply the comments I mentioned above:
1. model-format: optional, input model format type to be exported into the model archive.